dsn.go

```diff
@@ -335,6 +335,21 @@ func ParseDSN(dsn string) (cfg *Config, err error) {
 	return cfg, nil
 }
 
+// AddConnParameter adds a specified connection parameter to dsn
+func AddConnParameter(dsn string, key string, value interface{}) (string, error) {
```

Comment:
Do you want to expose this function to the users? If not, let's change the name start with a lower case.

Reply:
I think this is the only way to allow multistatement by the users -- we want them to manually specify the number of statements they want to execute, rigth?

Reply:
This function tweaks the DSN string, which should be created by:
- Manually create it from scratch by the user.
- Call `DSN` to convert from`Config` 

In the first case, it is up to the user about how to create it.
In the second case, you should consider adding the new parameter to `Config` or directly use `Params` to add it.

So I don't see why we need this function.

Reply:
I will delete this function.